common/src/main/java/net/opentsdb/utils/Comparators.java

```diff
@@ -60,4 +62,27 @@ public int compare(final Map<K, V> a, final Map<K, V> b) {
     }
     
   }
+
+  public static class ListComparison {
+
+    public static <T> boolean equalLists(List<T> one, List<T> two){
+      if (one == null && two == null){
+        return true;
+      }
+
+      if((one == null && two != null)
+              || one != null && two == null
+              || one.size() != two.size()){
+        return false;
+      }
+
+      //to avoid messing the order of the lists we will use a copy
+      one = new ArrayList<>(one);
+      two = new ArrayList<>(two);
```

Comment:
The idea of the copy here was to sort the arrays then walk one, calling `one.get(i).equals(two.get(i))` so we catch multiple entries of the same object in one or the other. Try adding some UTs for this one please.